lib/ukallocbbuddy/Config.uk

```diff
@@ -1,6 +1,16 @@
-config LIBUKALLOCBBUDDY
+menuconfig LIBUKALLOCBBUDDY
 	bool "ukallocbbuddy: Binary buddy page allocator"
 	default n
 	select LIBNOLIBC if !HAVE_LIBC
 	select LIBUKDEBUG
 	select LIBUKALLOC
+
+if LIBUKALLOCBBUDDY
+	config LIBUKALLOCBBUDDY_SMP_SAFE
+	bool "SMP safe allocator"
+	default n
+	depends on ARCH_ARM_64
```

Comment:
Should remove / change when we will have proper spinlock support on more architectures.